cmd/system-probe/config/config.go

```diff
@@ -202,6 +202,7 @@ func load(configPath string) (*Config, error) {
 	if !npmEnabled && usmEnabled {
 		log.Info("service_monitoring.enabled detected: enabling system-probe with network module running.")
 		c.EnabledModules[NetworkTracerModule] = struct{}{}
+		cfg.Set("network_config.enabled", true)
```

Comment:
This config value is intended to represent if NPM is enabled as a feature, not if the network tracer is running.

Reply:
Reverted, although the npm backend will drop events if this is not set.

Reply:
That was the intent. For folks who only have USM enabled, we want the backend to drop data in the NPM backend (after USM starts processing it).